Sources/Danger/GitHubDSL.swift

```diff
@@ -480,7 +480,7 @@ extension GitHub {
         }
 
         /// The id number of this milestone
-        public let id: Int // swiftlint:disable:this identifier_name
+        public let id: Int 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 483;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)